GoogleSheetI18n.Api.SimpleWebApi/ClientApp/src/custom.css

```diff
@@ -12,3 +12,7 @@ code {
   background-color: #1b6ec2;
   border-color: #1861ac;
 }
+
+.btn-validation {
```

Comment:
Pls, remove it as we can utilize default bootstrap classes.

Reply:
removed